Change the default binaries to more closely fit Debian policy
--- a/conf.c
+++ b/conf.c
@@ -263,8 +263,8 @@
 	for (i = 0; i < nitems(color_binds); i++)
 		c->color[i] = xstrdup(color_binds[i]);
 
-	conf_cmd_add(c, "lock", "xlock");
-	conf_cmd_add(c, "term", "xterm");
+	conf_cmd_add(c, "lock", "xscreensaver-command -lock");
+	conf_cmd_add(c, "term", "x-terminal-emulator");
 
 	(void)snprintf(c->known_hosts, sizeof(c->known_hosts), "%s/%s",
 	    homedir, ".ssh/known_hosts");
